internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -101,41 +100,77 @@ func (ws *workflows) StartWorkers(ctx context.Context) error {
 	return ws.worker.Start()
 }
 
-func (ws *workflows) StartWorkflow(ctx context.Context, session sdktypes.Session, opts StartWorkflowOptions) error {
-	sessionID := session.ID()
-
-	l := ws.l.Sugar().With("session_id", sessionID)
-
-	oid, err := ws.svcs.DB.GetOrgIDOf(ctx, session.ProjectID())
-	if err != nil {
-		return fmt.Errorf("get org id: %w", err)
-	}
-
+func memo(session sdktypes.Session, data sessiondata.Data) map[string]string {
 	memo := map[string]string{
-		"use_temporal_for_session_logs": strconv.FormatBool(opts.UseTemporalForSessionLogs),
-		"process_id":                    fixtures.ProcessID(),
-		"session_id":                    sessionID.String(),
-		"session_uuid":                  sessionID.UUIDValue().String(),
-		"deployment_id":                 session.DeploymentID().String(),
-		"deployment_uuid":               session.DeploymentID().UUIDValue().String(),
-		"project_id":                    session.ProjectID().String(),
-		"project_uuid":                  session.ProjectID().UUIDValue().String(),
-		"org_id":                        oid.String(),
-		"org_uuid":                      oid.UUIDValue().String(),
+		"process_id":      fixtures.ProcessID(),
+		"session_id":      session.ID().String(),
+		"session_uuid":    session.ID().UUIDValue().String(),
+		"deployment_id":   session.DeploymentID().String(),
+		"deployment_uuid": session.DeploymentID().UUIDValue().String(),
+		"project_id":      session.ProjectID().String(),
+		"project_uuid":    session.ProjectID().UUIDValue().String(),
+		"org_id":          data.OrgID.String(),
+		"org_uuid":        data.OrgID.UUIDValue().String(),
 	}
 
 	if session.ParentSessionID().IsValid() {
 		memo["parent_session_id"] = session.ParentSessionID().String()
 		memo["parent_session_uuid"] = session.ParentSessionID().UUIDValue().String()
 	}
 
-	maps.Copy(memo, session.Memo())
+	for k, v := range session.Memo() {
+		memo["session__"+k] = v
+	}
+
+	return memo
+}
+
+func (ws *workflows) StartChildWorkflow(wctx workflow.Context, childSession sdktypes.Session, parentSessionData *sessiondata.Data) (workflow.ChildWorkflowFuture, error) {
+	childSessionID := childSession.ID()
+
+	l := ws.l.Sugar().With("session_id", childSessionID)
```

Comment:
`Sugar()` isn't needed here, in fact it only makes line 157 worse

Reply:
done